pyuniqid/uniqid.py

```diff
@@ -1,32 +1,97 @@
-#!/usr/bin/env Python3
```

Comment:
Shebang line only matters if the file is going to be directly executed. This file really only gets imported, so having this shebang is unnecessary.